packages/react/src/themes/teams/components/Tree/treeTitleStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { ICSSInJSStyle } from '../../../types'
+import { pxToRem } from '../../../../lib'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+
+const treeTitleStyles = {
+  root: ({ variables, theme: { siteVariables } }): ICSSInJSStyle => ({
```

Comment:
```suggestion
  root: ({ variables: v, theme: { siteVariables } }): ICSSInJSStyle => ({
```

Let's use `v.*` naming